aws/Vault-Single-Deployment.json

```diff
@@ -878,7 +883,9 @@
          },
          "DependsOn":[
             "StoreMasterPassword",
-            "StoreAdminPassword"
+            "StoreAdminPassword",
```

Comment:
Add to all relevant templates this depend

Reply:
Done